### ISSUE TITLE
addPlayer doesn't work if player ped is frozen

### ISSUE THREAD

**Issue by minipunch** [meta-hub/meta_target#42]
is it just our death script or can anyone else confirm this also happens to them ?

it looks like it just doesn't detect the 'downed' person for some reason

our death script just sets their ped as just frozen with 1 health

**Comment by minipunch** [comment]
it appears `s2w.get(...)` doesn't detect entitys after they are 'frozen' via the `FreezeEntityPosition(...)`

**Comment by Taavi-AU** [comment]
ah right

**Comment by Taavi-AU** [comment]
thats odd, it should still return the player

**Comment by minipunch** [comment]
Yeah not sure myself why but it does appear that is the case

**Comment by minipunch** [comment]
Looks like that native `screenToWorld` uses doesn't register entities that are set to be 'frozen'

**Comment by Taavi-AU** [comment]
seems like it could be a fivem issue

**Comment by minipunch** [comment]
Yeah, or just how GTA V is meant to work I guess with that function lol...

Perhaps a workaround would be to just constantly set the entity's coords instead of using `FreezeEntityPosition`

I plan on trying that and can report back when done

Not sure how that might work if the player is being moved by someone else or put in to a vehicle or smth tho lol
